assets/training/vision/components/object_detection/spec.yaml

```diff
@@ -230,7 +230,7 @@ outputs:
 
 code: ../src
 
-environment: azureml://registries/azureml/environments/automl-dnn-vision-gpu/versions/1
+environment: azureml://registries/azureml/environments/automl-dnn-vision-gpu/versions/2
```

Comment:
same for this... update the environment version?

Reply:
I already updated to version 2. Could you please check the latest commits.